xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -40,43 +40,53 @@ using namespace EPG;
 
 CPVRTimerInfoTag::CPVRTimerInfoTag(bool bRadio /* = false */) :
   m_strTitle(g_localizeStrings.Get(19056)), // New Timer
-  m_strDirectory("/")
+  m_bFullTextEpgSearch(true)
```

Comment:
Why true as default? Probably most users will only record episodes based on title.
Maybe a setting for the default value?